src/services/infrastructure/url-generator/url.generator.service.ts

```
@@ -152,91 +77,32 @@ export class UrlGeneratorService {
-            LogContext.URL_GENERATOR
-          );
-        }
-        return subsubspaceUrlPath;
+        return await this.getSpaceUrlPathByAboutProfileID(profile.id);
       case ProfileType.USER:
-        const userEntityInfo = await this.getNameableEntityInfoOrFail(
+        const userEntityInfo = await this.getNameableEntityInfoForProfileOrFail(
           'user',
-          this.FIELD_PROFILE_ID,
           profile.id
         );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)